site/src/patterns/hero.md

```diff
@@ -36,8 +36,8 @@ Hero with a theme-aware companion image.
 <section
     class="hero hero-image background-color-body-accent gradient-border-right gradient-border-body-accent"
     style="
-        --hero-background-image-light: url('~/src/scaffold/media/hero-400-light.jpg');
-        --hero-background-image-dark: url('~/src/scaffold/media/hero-400-dark.jpg');
+        --hero-background-image-light: url('https://learn.microsoft.com/en-us/media/home-and-directory/home-hero_light.png?branch=main');
+        --hero-background-image-dark: url('https://learn.microsoft.com/en-us/media/home-and-directory/home-hero_dark.png');
```

Comment:
`?branch=main` is missed

Reply:
Actually, seems like it's not needed at all. Remove from the light image path then?

Reply:
updated